src/AspNet.Security.OAuth.Odnoklassniki/OdnoklassnikiAuthenticationConstants.cs

```diff
@@ -13,7 +13,9 @@ public static class OdnoklassnikiAuthenticationConstants
     {
         public static class Claims
         {
-            public const string ImageUrl = "urn:ok:profileimage";
+            public const string Pic1 = "urn:ok:pic1";
```

Comment:
I'd suggest naming these `Image1`, `Image2` and `Image3`, but otherwise this PR looks good now 😃

Reply:
One moment, please :)